vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+	test: {
+		include: ['tests/**/*.test.ts'],
+		environment: 'edge-runtime',
```

Comment:
I initially created two different config files one with `edge-runtime` to test the worker code and one with `nodejs` for the actual cli (and two different scripts), I then checked and the nodejs one wasn't really necessary (since in the tests I am mocking the node specific modules anyways) so I just figured that we could simplify things and just keep only this one. Please let me know if you think it's a bad idea.